dashboard/cron.py

```diff
@@ -154,8 +154,12 @@ def verify_course_ids(self):
 
             # error out when course id is invalid, otherwise add DataFrame to list
             if course_df.empty:
-                logger.error(f"""Course {course_id} don't have the entry in data warehouse yet. """)
-                invalid_course_id_list.append(course_id)
+                # Check if the course was ever locally updated. If it was updated it is invalid, otherwise don't consider this an error and skip it.
```

Comment:
```suggestion
                # Check if the course was ever updated by cron. If it was updated it is invalid, otherwise don't consider this an error and skip it.
```